Exception: referrer api.metrics.totals.initial_query is not part of Referrer Enum

### Self-Hosted Version

25.2.0

### CPU Architecture

x86_64

### Docker Version

25.0.8

### Docker Compose Version

2.34.0

### Machine Specification

- [x] My system meets the minimum system requirements of Sentry

### Steps to Reproduce

1. Run Sentry self-hosted
2. Randomly observe a crash (sorry, not sure how)

### Expected Result

No crash

### Actual Result

```sh
web-1  | worker 1 lifetime reached, it was running for 86401 second(s)
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:33:59 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:33:59 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_sessions.OrganizationSessionsEndpoint' response=200 user_id='670' is_app='False' token_type='None' is_fro
ntend_request='True' organization_id='4508776896724992' auth_id='None' path='/api/0/organizations/dephant/sessions/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=69548.61594462395 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests
='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | Traceback (most recent call last):
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 563, in send_envelope_wrapper
web-1  |     self._send_envelope(envelope)
web-1  |     ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 497, in _send_envelope
web-1  |     self._send_request(
web-1  |     ~~~~~~~~~~~~~~~~~~^
web-1  |         body.getvalue(),
web-1  |         ^^^^^^^^^^^^^^^^
web-1  |     ...<2 lines>...
web-1  |         envelope=envelope,
web-1  |         ^^^^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  |   File "/usr/src/sentry/src/sentry/utils/sdk.py", line 254, in patched_send_request
web-1  |     return _send_request(*args, **kwargs)
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 349, in _send_request
web-1  |     response = self._request(
web-1  |         "POST",
web-1  |     ...<2 lines>...
web-1  |         headers,
web-1  |     )
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 763, in _request
web-1  |     response = self._pool.request(
web-1  |         method,
web-1  |     ...<2 lines>...
web-1  |         headers=headers,  # type: ignore
web-1  |     )
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/interfaces.py", line 43, in request
web-1  |     response = self.handle_request(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection_pool.py", line 268, in handle_request
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection_pool.py", line 251, in handle_request
web-1  |     response = connection.handle_request(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 99, in handle_request
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 76, in handle_request
web-1  |     stream = self._connect(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 156, in _connect
web-1  |     stream = stream.start_tls(**kwargs)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_backends/sync.py", line 168, in start_tls
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_backends/sync.py", line 163, in start_tls
web-1  |     sock = ssl_context.wrap_socket(
web-1  |         self._sock, server_hostname=server_hostname
web-1  |     )
web-1  |   File "/usr/local/lib/python3.13/ssl.py", line 455, in wrap_socket
web-1  |     return self.sslsocket_class._create(
web-1  |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
web-1  |         sock=sock,
web-1  |         ^^^^^^^^^^
web-1  |     ...<5 lines>...
web-1  |         session=session
web-1  |         ^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  |   File "/usr/local/lib/python3.13/ssl.py", line 1067, in _create
web-1  |     self._sslobj = self._context._wrap_socket(
web-1  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
web-1  |         self, server_side, self.server_hostname,
web-1  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |         owner=self, session=self._session,
web-1  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  | TypeError: 'str' object is not callable
web-1  | 10:33:59 [ERROR] sentry_sdk.errors: Internal error in sentry_sdk
web-1  | panicked at /rustc/9fc6b43126469e3858e2fe86cafb4f0fd5068869/library/std/src/thread/local.rs:260:26:
web-1  |
web-1  | thread panicked while processing panic. aborting.
web-1  | worker 3 lifetime reached, it was running for 86401 second(s)
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:06 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_sessions.OrganizationSessionsEndpoint' response=200 user_id='670' is_app='False' token_type='None' is_fro
ntend_request='True' organization_id='4508776896724992' auth_id='None' path='/api/0/organizations/dephant/sessions/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=67395.22101783752 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests
='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | !!! uWSGI process 100289 got Segmentation Fault !!!
web-1  | *** backtrace of 100289 ***
web-1  | /.venv/lib/python3.13/site-packages/pyuwsgi.cpython-313-x86_64-linux-gnu.so(uwsgi_backtrace+0x2f) [0x7bf72107c31f]
web-1  | /.venv/lib/python3.13/site-packages/pyuwsgi.cpython-313-x86_64-linux-gnu.so(uwsgi_segfault+0x2f) [0x7bf72107c71f]
web-1  | /lib/x86_64-linux-gnu/libc.so.6(+0x3c050) [0x7bf721521050]
web-1  | /usr/local/lib/python3.13/lib-dynload/_ssl.cpython-313-x86_64-linux-gnu.so(+0x17a28) [0x7bf720585a28]
web-1  | /usr/local/lib/python3.13/lib-dynload/_ssl.cpython-313-x86_64-linux-gnu.so(+0x1c13c) [0x7bf72058a13c]
web-1  | /usr/local/lib/python3.13/lib-dynload/_ssl.cpython-313-x86_64-linux-gnu.so(+0x1c836) [0x7bf72058a836]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1d4e4d) [0x7bf72189ce4d]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(PyObject_Vectorcall+0x58) [0x7bf721852318]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(_PyEval_EvalFrameDefault+0x95bb) [0x7bf7218699fb]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1d925a) [0x7bf7218a125a]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1c6b23) [0x7bf72188eb23]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(_PyEval_EvalFrameDefault+0x57f9) [0x7bf721865c39]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1d925a) [0x7bf7218a125a]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1c6b23) [0x7bf72188eb23]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(_PyEval_EvalFrameDefault+0x57f9) [0x7bf721865c39]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1d92d6) [0x7bf7218a12d6]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(_PyEval_EvalFrameDefault+0x57f9) [0x7bf721865c39]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x1d92d6) [0x7bf7218a12d6]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x2a78a3) [0x7bf72196f8a3]
web-1  | /usr/local/bin/../lib/libpython3.13.so.1.0(+0x21d298) [0x7bf7218e5298]
web-1  | /lib/x86_64-linux-gnu/libc.so.6(+0x891c4) [0x7bf72156e1c4]
web-1  | /lib/x86_64-linux-gnu/libc.so.6(__clone+0x40) [0x7bf7215edac0]
web-1  | *** end of backtrace ***
web-1  | DAMN ! worker 1 (pid: 100288) MISTERIOUSLY killed by signal 6 :( trying respawn ...
web-1  | Respawned uWSGI worker 1 (new pid: 120394)
web-1  | DAMN ! worker 3 (pid: 100289) MISTERIOUSLY killed by signal 11 :( trying respawn ...
web-1  | Respawned uWSGI worker 3 (new pid: 120395)
web-1  | WSGI app 0 (mountpoint='') ready in 21 seconds on interpreter 0x7bf721bc0a70 pid: 120394 (default app)
web-1  | 10:39:22 [WARNING] django.request: Unauthorized: /api/0/relays/projectconfigs/ (status_code=401 request=<WSGIRequest: POST '/api/0/relays/projectconfigs/?version=3'>)
web-1  | 10:39:22 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.011791467666625977 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:39:22 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.011563301086425781 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:39:22 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.01037144660949707 rate_limit_type='DNE' concur
rent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:39:22 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.0034637451171875 rate_limit_type='DNE' concurr
ent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:39:22 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.005441904067993164 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | WSGI app 0 (mountpoint='') ready in 14 seconds on interpreter 0x7bf721bc0a70 pid: 120395 (default app)
web-1  | 10:39:58 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.006652355194091797 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:40:58 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.005151510238647461 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:41:44 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='True' organization_id='
None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_lim
it_category='None' request_duration_seconds=0.0038008689880371094 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:41:44 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.auth_login.AuthLoginView' response=200 user_id='None' is_app='None' token_type='None' is_frontend_request='True' organi
zation_id='None' auth_id='None' path='/auth/login/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_li
mited='False' rate_limit_category='None' request_duration_seconds=0.10272455215454102 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='
None')
web-1  | 10:41:47 [INFO] sentry.access.api: api.access (method='POST' view='sentry.web.frontend.auth_login.AuthLoginView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='True' organ
ization_id='None' auth_id='None' path='/auth/login/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_l
imited='False' rate_limit_category='None' request_duration_seconds=0.024647235870361328 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining
='None')
web-1  | 10:41:47 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.auth_organization_login.AuthOrganizationLoginView' response=200 user_id='None' is_app='None' token_type='None' is_front
end_request='True' organization_id='None' auth_id='None' path='/auth/login/code-ops/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrom
e/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.03836631774902344 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' gro
up='None' limit='None' remaining='None')
web-1  | 10:41:48 [INFO] sentry.access.api: api.access (method='POST' view='sentry.web.frontend.auth_organization_login.AuthOrganizationLoginView' response=302 user_id='None' is_app='None' token_type='None' is_fron
tend_request='True' organization_id='None' auth_id='None' path='/auth/login/code-ops/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chro
me/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.028957843780517578 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' g
roup='None' limit='None' remaining='None')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.needs-validation (DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL=False self_hosted=True)
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.logged-in (ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.auth: user.auth.success (ip_address='172.19.0.28' username='ercan.ucan' organization_id=2)
web-1  | 10:41:50 [INFO] sentry.auth: sso.login-pipeline.attach-existing-identity (wipe_result='None' organization_id=2 user_id=236 auth_identity_user_id=236 auth_provider_id=1 idp_identity_id=MigratingIdentityId(i
d='13868', legacy_id='ercan.ucan@siemens.com') idp_identity_email='ercan.ucan@siemens.com')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.needs-validation (DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL=False self_hosted=True)
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.logged-in (ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.auth: user.auth.success (ip_address='172.19.0.28' username='ercan.ucan' organization_id=2)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.auth_provider_login.AuthProviderLoginView' response=302 user_id='236' is_app='None' token_type='None' is_frontend_reque
st='True' organization_id='None' auth_id='None' path='/auth/sso/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/5
37.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.811924934387207 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None'
 remaining='None')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/' method='GET' ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='236' is_app='False' token_type='None' is_frontend_request='True' organization_id='
None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_lim
it_category='None' request_duration_seconds=0.06149172782897949 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/organizations/core/issues/' method='GET' ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.web.frontend.base: access.must-sso (organization_id=42 user_id=236)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.react_page.ReactPageView' response=302 user_id='236' is_app='False' token_type='None' is_frontend_request='True' organi
zation_id='None' auth_id='None' path='/organizations/core/issues/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/
537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.05766463279724121 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='No
ne' remaining='None')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/auth/login/core/?next=%2Forganizations%2Fcore%2Fissues%2F' method='GET' ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.auth_organization_login.AuthOrganizationLoginView' response=200 user_id='236' is_app='False' token_type='None' is_front
end_request='True' organization_id='None' auth_id='None' path='/auth/login/core/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/13
5.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.1254270076751709 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None' group='N
one' limit='None' remaining='None')
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/api/0/organizations/core/?detailed=0&include_feature_flags=1' method='GET' ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/api/0/organizations/core/projects/?all_projects=1&collapse=latestDeploys&collapse=unusedFeatures' method='GET' ip_address=
'172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.superuser: superuser.request (url='http://example.sentry.com/api/0/organizations/core/teams/' method='GET' ip_address='172.19.0.28' user_id=236)
web-1  | 10:41:50 [INFO] sentry.api.base: access.must-sso (organization_id=42 user_id=236)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_projects.OrganizationProjectsEndpoint' response=401 user_id='236' is_app='False' token_type='None' is_fro
ntend_request='True' organization_id='None' auth_id='None' path='/api/0/organizations/core/projects/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, l
ike Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.10157990455627441 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset
_time='None' group='None' limit='None' remaining='None')
web-1  | 10:41:50 [WARNING] django.request: Unauthorized: /api/0/organizations/core/projects/ (status_code=401 request=<WSGIRequest: GET '/api/0/organizations/core/projects/?all_projects=1&collapse=latestDeploys&co
llapse=unusedFeatures'>)
web-1  | 10:41:50 [INFO] sentry.api.base: access.must-sso (organization_id=42 user_id=236)
web-1  | 10:41:50 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_details.OrganizationDetailsEndpoint' response=401 user_id='236' is_app='False' token_type='None' is_front
end_request='True' organization_id='None' auth_id='None' path='/api/0/organizations/core/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko)
Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.11934542655944824 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests='None' reset_time='None
' group='None' limit='None' remaining='None')
web-1  | 10:41:50 [WARNING] django.request: Unauthorized: /api/0/organizations/core/ (status_code=401 request=<WSGIRequest: GET '/api/0/organizations/core/?detailed=0&include_feature_flags=1'>)
web-1  | 10:32:58 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.00683140754699707 rate_limit_type='DNE' concur
rent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | 10:33:58 [INFO] sentry.access.api: api.access (method='GET' view='sentry.web.frontend.home.HomeView' response=302 user_id='None' is_app='None' token_type='None' is_frontend_request='False' organization_id=
'None' auth_id='None' path='/' caller_ip='172.19.0.28' user_agent='Blackbox Exporter/0.26.0' rate_limited='False' rate_limit_category='None' request_duration_seconds=0.004502534866333008 rate_limit_type='DNE' concu
rrent_limit='None' concurrent_requests='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | worker 1 lifetime reached, it was running for 86401 second(s)
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:33:59 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:33:59 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_sessions.OrganizationSessionsEndpoint' response=200 user_id='670' is_app='False' token_type='None' is_fro
ntend_request='True' organization_id='4508776896724992' auth_id='None' path='/api/0/organizations/dephant/sessions/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=69548.61594462395 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests
='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | Traceback (most recent call last):
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 563, in send_envelope_wrapper
web-1  |     self._send_envelope(envelope)
web-1  |     ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 497, in _send_envelope
web-1  |     self._send_request(
web-1  |     ~~~~~~~~~~~~~~~~~~^
web-1  |         body.getvalue(),
web-1  |         ^^^^^^^^^^^^^^^^
web-1  |     ...<2 lines>...
web-1  |         envelope=envelope,
web-1  |         ^^^^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  |   File "/usr/src/sentry/src/sentry/utils/sdk.py", line 254, in patched_send_request
web-1  |     return _send_request(*args, **kwargs)
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 349, in _send_request
web-1  |     response = self._request(
web-1  |         "POST",
web-1  |     ...<2 lines>...
web-1  |         headers,
web-1  |     )
web-1  |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/transport.py", line 763, in _request
web-1  |     response = self._pool.request(
web-1  |         method,
web-1  |     ...<2 lines>...
web-1  |         headers=headers,  # type: ignore
web-1  |     )
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/interfaces.py", line 43, in request
web-1  |     response = self.handle_request(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection_pool.py", line 268, in handle_request
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection_pool.py", line 251, in handle_request
web-1  |     response = connection.handle_request(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 99, in handle_request
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 76, in handle_request
web-1  |     stream = self._connect(request)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_sync/connection.py", line 156, in _connect
web-1  |     stream = stream.start_tls(**kwargs)
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_backends/sync.py", line 168, in start_tls
web-1  |     raise exc
web-1  |   File "/.venv/lib/python3.13/site-packages/httpcore/_backends/sync.py", line 163, in start_tls
web-1  |     sock = ssl_context.wrap_socket(
web-1  |         self._sock, server_hostname=server_hostname
web-1  |     )
web-1  |   File "/usr/local/lib/python3.13/ssl.py", line 455, in wrap_socket
web-1  |     return self.sslsocket_class._create(
web-1  |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
web-1  |         sock=sock,
web-1  |         ^^^^^^^^^^
web-1  |     ...<5 lines>...
web-1  |         session=session
web-1  |         ^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  |   File "/usr/local/lib/python3.13/ssl.py", line 1067, in _create
web-1  |     self._sslobj = self._context._wrap_socket(
web-1  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~^
web-1  |         self, server_side, self.server_hostname,
web-1  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |         owner=self, session=self._session,
web-1  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
web-1  |     )
web-1  |     ^
web-1  | TypeError: 'str' object is not callable
web-1  | 10:33:59 [ERROR] sentry_sdk.errors: Internal error in sentry_sdk
web-1  | panicked at /rustc/9fc6b43126469e3858e2fe86cafb4f0fd5068869/library/std/src/thread/local.rs:260:26:
web-1  |
web-1  | thread panicked while processing panic. aborting.
web-1  | worker 3 lifetime reached, it was running for 86401 second(s)
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.initial_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.totals.second_query is not part of Referrer Enum
web-1  | Traceback (most recent call last):
web-1  |   File "/usr/src/sentry/src/sentry/snuba/referrer.py", line 997, in validate_referrer
web-1  |     raise Exception(error_message)
web-1  | Exception: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:05 [WARNING] sentry.snuba.referrer: referrer api.metrics.series.second_query is not part of Referrer Enum
web-1  | 10:34:06 [INFO] sentry.access.api: api.access (method='GET' view='sentry.api.endpoints.organization_sessions.OrganizationSessionsEndpoint' response=200 user_id='670' is_app='False' token_type='None' is_fro
ntend_request='True' organization_id='4508776896724992' auth_id='None' path='/api/0/organizations/dephant/sessions/' caller_ip='172.19.0.28' user_agent='Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36' rate_limited='False' rate_limit_category='None' request_duration_seconds=67395.22101783752 rate_limit_type='DNE' concurrent_limit='None' concurrent_requests
='None' reset_time='None' group='None' limit='None' remaining='None')
web-1  | !!! uWSGI process 100289 got Segmentation Fault !!!
web-1  | *** backtrace of 100289 ***
```

### Event ID

_No response_

--- a/src/sentry/snuba/referrer.py
+++ b/src/sentry/snuba/referrer.py
@@ -116,6 +116,7 @@ class Referrer(StrEnum):
     API_ISSUES_ISSUE_EVENTS = "api.issues.issue_events"
     API_ISSUES_RELATED_ISSUES = "api.issues.related_issues"
     API_METRICS_TOTALS = "api.metrics.totals"
+    API_METRICS_TOTALS_INITIAL_QUERY = "api.metrics.totals.initial_query"
     API_ORGANIZATION_TRACE_ITEM_DETAILS = "api.organization-trace-item-details"
     API_ORGANIZATION_EVENT_STATS_FIND_TOPN = "api.organization-event-stats.find-topn"
     API_ORGANIZATION_EVENT_STATS_METRICS_ENHANCED = "api.organization-event-stats.metrics-enhanced"
@@ -396,6 +397,9 @@ class Referrer(StrEnum):
     API_PERFORMANCE_TRANSACTION_NAME_SEARCH_BAR = "api.performance.transaction-name-search-bar"
     API_PERFORMANCE_TRANSACTION_SPANS = "api.performance.transaction-spans"
     API_PERFORMANCE_TRANSACTION_SUMMARY_DURATION = "api.performance.transaction-summary.duration"
+    API_PERFORMANCE_TRANSACTION_SUMMARY_METRICS_ENHANCED_PRIMARY = (
+        "api.performance.transaction-summary.metrics-enhanced.primary"
+    )
     API_PERFORMANCE_TRANSACTION_SUMMARY_SIDEBAR_CHART_METRICS_ENHANCED = (
         "api.performance.transaction-summary.sidebar-chart.metrics-enhanced"
     )

